From ac849e7960547d418009a783da654d5917dbfe2d Mon Sep 17 00:00:00 2001
From: Sascha Grunert <sgrunert@redhat.com>
Date: Fri, 16 Jul 2021 12:13:36 +0200
Subject: [PATCH] syscalls: add close_range() syscall

The syscall has been added a while ago so we should support resolving
it, too.

Signed-off-by: Sascha Grunert <sgrunert@redhat.com>
Reviewed-by: Tom Hromatka <tom.hromatka@oracle.com>
[PM: subject line tweak]
Signed-off-by: Paul Moore <paul@paul-moore.com>
(imported from commit 01e5750e7c84bb14e5a5410c924bed519209db06)
---
 include/seccomp-syscalls.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/seccomp-syscalls.h b/include/seccomp-syscalls.h
index 7b69214c..1ca500be 100644
--- a/include/seccomp-syscalls.h
+++ b/include/seccomp-syscalls.h
@@ -476,6 +476,8 @@
 
 #define __SNR_close			__NR_close
 
+#define __SNR_close_range		__NR_close_range
+
 #ifdef __NR_connect
 #define __SNR_connect			__NR_connect
 #else
